verteiler/src/test/java/io/datanerds/verteiler/ProcessorTest.java

```diff
@@ -80,4 +80,16 @@ public void processMessageException() throws Exception {
         verify(relay, never()).setOffset(record);
     }
 
+    @Test
+    public void stoppedSetOnException() throws Exception {
+        final Processor<Integer, String> processor = new Processor<>(new TopicPartition("Hello", 1), relay, message -> {
```

Comment:
Defensive programming, protection against reassignment of variables, blah blah ;)